priv/schema/rabbit.schema

```diff
@@ -1086,9 +1086,8 @@ end}.
     {datatype, {enum, [true, false]}}
 ]}.
 
-{mapping, "log.syslog.ip", "syslog.dest_host", [
```

Comment:
Can we leave `log.syslog.ip` in place for compatibility? (it also makes some sense as an alias anyway)

Reply:
I wanted to do that, but since they both set `syslog.dest_host`, it doesn't work.

Reply:
Oh, OK.

Reply:
Hm there may be options if the value can be transformed, maybe. Let me check.